tools/src/gencerts/configuration.yaml.example

```diff
@@ -0,0 +1,21 @@
+dev:
```

Comment:
Should we generate a custom config in connect-to-cluster instead of using `lib/configuration.yaml`? that can have some override parameters for altDNS, clients, expiration, etc?

Reply:
That would work, the tool doesn't make any assumption about the global shape of the config file so you could have a detached one with only the required fields for TLS.